api/enki.py

```diff
@@ -421,11 +366,18 @@ def fulfill(self, patron, pin, licensepool, internal_format):
                 self.log.error("User validation against Enki server with %s / %s was unsuccessful."
                     % (patron.authorization_identifier, pin))
                 raise AuthorizationFailedException()
-        drm_type = self.get_enki_drm_type(book_id)
```

Comment:
There's no reason to make another HTTP request to get the DRM type -- it's always the type associated with the DeliveryMechanism.